module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,17 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <string>
```

Comment:
You have included string in the header,so you do not need to include it here.

Reply:
+1